server/reports/item-usage/2_5/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "is_custom": false,
-  "version": "2.3.0",
+  "version": "2.5.0",
```

Comment:
Noting here that 2.5.0 version change is required because stockOnHand in itemStatsNode was only included in 2.5 release